Robot Code/src/org/usfirst/frc/team3316/robot/robotIO/Actuators.java

```diff
@@ -53,9 +53,12 @@
 	// Spare
 	public DBugSpeedController spareMotor;
 	public SpeedController spareMotorSC;
-
-	public Actuators()
+	
+	public Actuators() {}
+	
+	public void AllActuators()
```

Comment:
Looks like this was your method to keep the original code intact. You're using git, there's no reason to do that. Finish the changes as they should run on the robot.